src/app/combo/combo.sample.ts

```diff
@@ -410,4 +418,14 @@ export class ComboSampleComponent implements OnInit, AfterViewInit {
             }
         );
     }
+
+    public openOnClearSelectionChanging(event: ISimpleComboSelectionChangingEventArgs){
```

Comment:
Create an overload for the `handleSelectionChange` function instead and while you are at it - rename it to `handleSelectionChanging`.

Reply:
@jackofdiamond5 In fact, there is already a `handleSelectionChanging ` function that is handled by two other components and performs an action. Maybe I should use the `handleSelectionChange ` function which currently only prints the console event or should I remove this function and leave the added `openOnClearSelectionChanging` ?

Reply:
Well, create overloads for the `handleSelectionChanging` method and then get rid of the `handleSelectionChange` method since it's supposed to be doing the same thing - handle the `selectionChanging` event.